Lib/ftplib.py

```diff
@@ -93,8 +92,7 @@ class FTP:
     below for details).
     The download/upload functions first issue appropriate TYPE
     and PORT or PASV commands.
-    '''
-
+    """
```

Comment:
Changing the single quotes to double quotes is just a formatting change, these can remain the same.